plugins/bulk-import/src/components/Repositories/RepositoriesListColumns.tsx

```diff
@@ -41,19 +53,24 @@ export const columns: TableColumn<AddRepositoriesData>[] = [
     field: 'catalogInfoYaml.status',
     type: 'string',
     align: 'left',
+    customSort: (a: AddRepositoryData, b: AddRepositoryData) =>
+      descendingComparator(a, b, 'catalogInfoYaml.status'),
+    render: (data: AddRepositoryData) =>
+      getImportStatus(data.catalogInfoYaml?.status as string),
   },
   {
     title: 'Last updated',
-    field: 'lastUpdated',
+    field: 'catalogInfoYaml.lastUpdated',
     type: 'string',
```

Comment:
Show a formatted date:

```suggestion
    type: 'datetime',
```

Reply:
mark it as resolved to merge this PR, @debsmita1 will check if we can address this in a follow up PR.